app/models/ranker/highest_strength_of_schedule.rb

```diff
@@ -0,0 +1,16 @@
+module Ranker
+  class HighestStrengthOfSchedule < Base
+    def self.apply_stats(row, all_rows, _)
+      sos = 0
+      row.player.games.each do |game|
+        opponent = game.opponent(row.player)
+        sos += all_rows[opponent.id].result_points
+      end
+      row.add_stat(:sos, sos)
```

Comment:
Think I agree. I was actually too lazy to implement it, but I think I will in the near future. :+1: